src/patternfly/components/Badge/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+:root {
+  --pf-c-badge--BorderRadius: var(--pf-global--BorderRadius);
+  --pf-c-badge--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-badge--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-badge--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-badge--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-badge--m-read--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-badge--m-read--Color: var(--pf-global--dark--Color);
```

Comment:
I think this should be `--pf-global--Color--dark-100`

Reply:
Ah you're right. That was in the component generator and I forgot to update it. I'll also update the component generator to be the default color.